layers/meta-balena-raspberrypi/recipes-core/images/balena-image.bbappend

```diff
@@ -12,8 +12,10 @@ SDIMG_KERNELIMAGE:raspberrypi3-64 ?= "kernel8.img"
 SDIMG_KERNELIMAGE:raspberrypi0-2w-64 ?= "kernel8.img"
 
 # Increase Root File system size
-# ToDo (Waqar): Find correct size and fix issues
-IMAGE_ROOTFS_SIZE:revpi-connect-s ?= "2785280"
+IMAGE_ROOTFS_SIZE:revpi-connect-s ?= "319488"
+IMAGE_OVERHEAD_FACTOR:revpi-connect-s ?= "1.0"
+IMAGE_ROOTFS_EXTRA_SPACE:revpi-connect-s ?= "53248"
```

Comment:
This variable is particularly useful when we want to ensure that a specific amount of free disk space is available on a device after an image is installed and running